index.js

```diff
@@ -1,16 +1,35 @@
-import {HTML5Video, Log, Events} from 'clappr'
-import shaka from 'shaka-player'
+const {HTML5Video, Log, Events} = window.Clappr;
+const shaka = window.shaka;
```

Comment:
yes that was a quick fix to make my build go through.

i agree modules is better and then eventually telling your toolchain to take that as external dep.